sentry.config.js

```diff
@@ -25,6 +25,24 @@ export const sentryOptions = {
     // Common non-actionable errors
     "rejected transaction",
     "User closed modal",
+    "Loading chunk",
+    "Failed to execute '",
+    "NetworkError when attempting to fetch resource.",
+    "googlefc is not defined",
+    "__cmp is not defined",
+    "Cannot read properties of undefined (reading 'cmp')",
+    "Cannot read properties of undefined (reading 'loaded')",
```

Comment:
this means we would even ignore errors that happen if *we* try to access `.loaded` on something undefined so this is a bit too scary to exclude imo